src/main/java/com/revellr/core/partyservice/Models/User.java

```diff
@@ -0,0 +1,60 @@
+package com.revellr.core.partyservice.Models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User{
+    @Id
+    @GeneratedValue(strategy=GenerationType.AUTO)
+
+    private Integer id;
+
+    private String firstName;
+
+    private String lastName;
+
+    private String userName;
+
+    private String mobileNumber;
+
+    private String email;
+
+    private String ImageURI;
+
+    private String CurrentOrganisation;
+
+    private String College;
+
+
+    public Integer getId() {
```

Comment:
Getters and Setters not needed as they would be generated by lombok with annotations